bio_hansel/scripts/find_cluster.py

```diff
@@ -0,0 +1,100 @@
+from typing import List, Dict
+
+import numpy as np
+import pandas as pd
+import scipy as sp
+
+from scipy.cluster.hierarchy import fcluster, linkage
+from scipy.spatial.distance import pdist
+
+
+def find_clusters(df: pd.DataFrame, min_threshold: float,
+                  max_threshold: float) -> Dict[str, str]:
+    """
+    Takes in a vcf file and creates clusters from the scipy hierarchy clustering algorithm
+    
+    Example:
+    '/path/example.vcf' -> {'mysnpsSRR2323':'1', 'mysnpsSRR232323':'2'}
+    
+    Args:
+        df: contains the vcf information in the DataFrame format
+        min_threshold: minimum threshold for the flat cluster array
+        max_threshold: maximum threshold for the flat cluster array
+
+    Returns:
+        cluster_dict: a dictionary indicating the cluster membership of each of the supplied genomes in
+                the vcf file
+    """
+
+    distance_matrix = compute_distance_matrix(df)
+    clustering_array = create_linkage_array(distance_matrix)
+
+    flat_clusters = output_flat_clusters(clustering_array, df.columns,
+                                         min_threshold, max_threshold)
+
+    subset = flat_clusters.iloc[0]
```

Comment:
So there may be many columns of flat clusters derived at different threshold levels, but you're only using the first value? Also are you sure you're getting a `Dict[str, str]` when you call `to_dict()`?